packages/amplify-cli-core/src/overrides-manager/override-skeleton-generator.ts

```diff
@@ -0,0 +1,81 @@
+import fs from 'fs-extra';
+import { $TSContext, getPackageManager } from '../index';
+import execa from 'execa';
+import * as path from 'path';
+import { printer } from 'amplify-prompts';
+import { JSONUtilities } from '../jsonUtilities';
+
+export const generateOverrideSkeleton = async (context: $TSContext, srcResourceDirPath: string, destDirPath: string): Promise<void> => {
+  // 1. Create skeleton package
+  const backendDir = context.amplify.pathManager.getBackendDirPath();
+  const overrideFile = path.join(destDirPath, 'override.ts');
```

Comment:
Can the filename be kept in a centralized location ? in amplify-meta or config files.

Reply:
override file path will be different for each category/resource , that's why I make it dependent on destPath which will be called by each category transform when applying override